README.md

```diff
@@ -73,17 +73,11 @@ $ cargo build --release
 To connect NEAR Indexer for Explorer to the specific chain you need to have necessary configs, you can generate it as follows:
 
 ```bash
-$ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+$ ./target/release/indexer-explorer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
```

Comment:
Nice catch, but I'm not sure. @frol what do you think maybe it's better to change the name in `Cargo.toml` instead?

Reply:
@khorolets I would rename it in Cargo.toml to `near-indexer-for-explorer` to match the name of the repo and avoid confusion

Reply:
Ok, i discarder there changes. Should I do one more PR with changing `Cargo.toml`?

Reply:
@OS-M Yes, please, create a separate PR with the change in Cargo.toml and README